.travis.yml

```diff
@@ -16,7 +8,32 @@ branches:
 before_install:
   - pip install --upgrade pip setuptools wheel
 
-install:
-  - pip install -r ${REQS}
+matrix:
+  include:
+  - os: linux
+    env: NAME=mypy
+    python: "3.5"
+    install:
+      - cat dev-requirements.txt | grep mypy | xargs pip install
+    script: check/mypy
+
+  - os: linux
+    env: NAME=pylint
+    python: "3.5"
+    install:
+      - cat dev-requirements.txt | grep pylint | xargs pip install
+    script: check/pylint
 
-script: continuous-integration/travis.sh "${MODE}"
+  - os: linux
+    env: NAME=pytest-and-incremental-coverage
+    python: "3.5"
+    install:
+      - pip install -r dev-requirements.txt
+    script: check/pytest-and-incremental-coverage master
```

Comment:
Can we make the comparison branch optional and default to master?

Reply:
I couldn't decide between master, origin/master, and upstream/master.

I guess I could do them in some particular order, and also compute the merge base so that I had the latest one.